### PR TITLE
Add environment settings

### PR DESCRIPTION
**Pull request by marceloschmidt**
Adds a setting to set the dialogflow environment

### CODE REVIEW

File: config/Settings.ts [hunk 1]
```diff
@@ -38,10 +39,18 @@ export const settings: Array<ISetting> = [
         id: AppSetting.DialogflowProjectId,
         public: true,
         type: SettingType.STRING,
-        packageValue: '',
+        packageValue: 'draft',
```
Comment (Shailesh351) [inline config/Settings.ts:42]:
```suggestion
        packageValue: '',
```

File: config/Settings.ts [hunk 2]
```diff
@@ -38,10 +39,18 @@ export const settings: Array<ISetting> = [
...
         i18nLabel: 'dialogflow_project_id',
         required: true,
     },
+    {
+        id: AppSetting.DialogflowEnvironment,
+        public: true,
+        type: SettingType.STRING,
+        packageValue: '',
```
Comment (Shailesh351) [inline config/Settings.ts:50]:
```suggestion
        packageValue: 'draft',
```

**Comment by marceloschmidt** [comment]
Thanks @Shailesh351

**Comment by marceloschmidt** [comment]
@murtaza98 could you please review this? 
I am facing an issue: when I upload this version to my local instance, it does not create the new setting. Any insights on it?

**Comment by murtaza98** [comment]
> @murtaza98 could you please review this? I am facing an issue: when I upload this version to my local instance, it does not create the new setting. Any insights on it?

@marceloschmidt I tried to install it on my server and it worked, so no issue with the code. So maybe try reinstalling the app. Hopefully, that should fix it. Let me know if the issue still persists

**Review by murtaza98 — APPROVED** [review]

**Pull request merged**